### PR TITLE
OSDOCS-1977 - Categorising installation methods by provisioning type

### PR DESCRIPTION
**Pull request by pneedle-rh**
This applies to master and branch/enterprise-4.8.

The PR categorises the installation methods in the "Preparing to install on..." assemblies into installer-provisioned and user-provisioned methods. It also adds a brief description about installer-provisioned and user-provisioned installations and includes a link to further reading on that topic.

This relates to https://issues.redhat.com/browse/OSDOCS-1977.

Here are the previews:

- [Preparing to install on AWS](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/preparing-to-install-on-aws.html)
- [Preparing to install on Azure](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure/preparing-to-install-on-azure.html)
- [Preparing to install on GCP](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/preparing-to-install-on-gcp.html)
- [Preparing for bare metal cluster installation](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/preparing-to-install-on-bare-metal.html)
- [Preparing to install with z/VM on IBM Z and LinuxONE](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/preparing-to-install-on-ibm-z.html)
- [Preparing to install with RHEL KVM on IBM Z and LinuxONE](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/preparing-to-install-on-ibm-z-kvm.html)
- [Preparing to install on IBM Power Systems](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/preparing-to-install-on-ibm-power.html)
- [Preparing to install on OpenStack](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/preparing-to-install-on-openstack.html)
- [Preparing to install on RHV](https://deploy-preview-33770--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/preparing-to-install-on-rhv.html)

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Thanks!

**Comment by pneedle-rh** [comment]
@pamoedom can you please review `installing/installing_openstack/preparing-to-install-on-openstack.adoc` specifically, from a QE perspective? In particular, is the wording at the start of the new subsections accurate for OpenStack installer-provisioned and user-provisioned installations? Thanks!

File: installing/installing_aws/preparing-to-install-on-aws.adoc [hunk 1]
```diff
@@ -21,7 +21,14 @@ If the cloud identity and access management (IAM) APIs are not accessible in you
...
 
-You can install {product-title} on AWS by using one of the following deployment methods.
+You can install {product-title} on installer-provisioned or user-provisioned infrastructure. The default installation type uses installer-provisioned infrastructure, where the installation program provisions the underlying infrastructure for the cluster. You can also install {product-title} on infrastructure that you provision. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
+
+See xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+
+[id="choosing-an-method-to-install-ocp-on-aws-installer-provisioned"]
+=== Installing a cluster on installer-provisioned infrastructure
+
+You can install a cluster on AWS infrastructure that is provisioned by the {product-title} installation program, by using one of the following methods.
```
Comment (bergerhoffer) [inline installing/installing_aws/preparing-to-install-on-aws.adoc:31]: I'd suggest using a colon to end each of these sentences that leads into the bulleted lists.

**Review by bergerhoffer — COMMENTED** [review]
Just one minor suggestion.

File: installing/installing_aws/preparing-to-install-on-aws.adoc (hunk 1 above)
Comment (pneedle-rh) [reply installing/installing_aws/preparing-to-install-on-aws.adoc]: @bergerhoffer thank you for your review and for this suggestion. I have now resolved this in each of the assemblies.

**Comment by pneedle-rh** [comment]
@rlobillo can you please review the `installing/installing_openstack/preparing-to-install-on-openstack.adoc` specifically, from a QE perspective? In particular, is the wording at the start of the new subsections accurate for OpenStack installer-provisioned and user-provisioned installations? Thanks!

**Pull request merged**